examples/introduction_script.py

```diff
@@ -15,7 +15,7 @@
 
 # Create empty Structure object
 
-mdl = Structure(name='introduction', path='/home/al/temp/')
+mdl = Structure(name='introduction', path='C:/temp/')
```

Comment:
would be good if we could make this non-specific to windows

Reply:
not sure how though...this is the folder where all the analysis stuff happens. If I put `compas_fea.TEMP` then a bunch of crap is created in the rhino location (or gives me an error)